scripts/BuildAndTest.ps1

```diff
@@ -14,6 +14,8 @@
     Do not rebuild the SARIF object model from the schema.
 .PARAMETER NoBuild
     Do not build.
+.PARAMETER NoBuildSample
```

Comment:
>NoBuildSample [](start = 11, length = 13)

Why do we need this knob? Is there really a situation where disabling the sample build is of material interest? Who would discover this knob in any case? :)

If you want something like this, you should consider a setting for 'Comprehensive' or 'FullGate' and then have things like this disabled by default for happy path developer use. Those settings make sense 'fast and most relevant pre-check-in' and 'slow and comprehensive, verifies that the full VSTS build can do its thing'.

Reply:
I have a switch like this for every step in BuildAndTest. I agree it's not best. I'll do something like you suggest in a separate PR.

Re: discoverability, it's the same as any PowerShell comment that uses the standard doc comment header:
```
Get-Help .\BuildAndTest.ps1 -Detailed
```

Reply:
Roger.

---
In reply to: [211004442](https://github.com/Microsoft/sarif-sdk/pull/1024#discussion_r211004442) [](ancestors = 211004442)

Reply:
@michaelcfanning Come to think of it, I use these knobs all the time when testing changes to the script. But yes, there's probably a better way.